backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -730,7 +731,8 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
     * @return The Try wrapped and mapped WOM value.
     */
   final def outputValueMapper(womValue: WomValue): Try[WomValue] = {
-    WomFileMapper.mapWomFiles(mapOutputWomFile)(womValue)
+    // TODO: is this inputsToNot*DE*Localize?
```

Comment:
why would inputs be delocalized?

Reply:
oops - this should really be `outputsToNotDelocalize`, and since the `inputsToNotDelocalize` is not relevant to that, I think this should probably just hard-code `Set.empty` for now